model_configs/dna_tuning_config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "population_size": 2,
+    "n_generations": 10,
+    "output_path": "./tuning_results",
+    "attributes": {
+        "hidden_layer_size": {
+            "int": [1, 512, 1]
+        },
+        "n_hidden_layers": {
+            "int": [1, 10, 1]
+        },
+        "use_batch_norm": {
+            "bool": []
```

Comment:
using the above suggestion, this could become:
```
"use_batch_norm": {
    "type": bool
}
```
and then we don't have this odd looking `"bool": []`